processor/logdedupprocessor/README.md

```diff
@@ -25,13 +25,17 @@ This processor is used to deduplicate logs by detecting identical logs over a ra
-| log_count_attribute     | string | `log_count`    | The name of the count attribute of deduplicated logs that will be added to the emitted aggregated log. |
-| timezone     | string | `UTC`    | The timezone of the `first_observed_timestamp` and `last_observed_timestamp` timestamps on the emitted aggregated log. The available locations depend on the local IANA Time Zone database. [This page](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) contains many examples, such as `America/New_York`.  |
-| exclude_fields     | []string | `[]`    | Fields to exclude from duplication matching. Fields can be excluded from the log `body` or `attributes`. These fields will not be present in the emitted aggregated log. Nested fields must be `.` delimited. If a field contains a `.` it can be escaped by using a `\` see [example config](#example-config-with-excluded-fields).<br><br>**Note**: The entire `body` cannot be excluded. If the body is a map then fields within it can be excluded. |
+| Field               | Type     | Default     | Description                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| ---                 | ---      | ---         | ---                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| interval            | duration | `10s`       | The interval at which logs are aggregated. The counter will reset after each interval.                                                                                                                                                                                                                                                                                                                                                                  |
+| condition           | string   | `true`      | An [OTTL] expression used to match which log records to sample from. All paths in the [log context] are available to reference. All [converters] are available to use.                                                                                                                                                                                                                                                                                  |
```

Comment:
Intuitively, as a user I think I would expect a condition to control whether or not a log is _deduplicated_, not whether it is kept at all. In other words, I would have expected that logs that do not match the condition pass through the processor without being affected.

The actual meaning seems to be that the condition acts as a filter for the logs. The problem is, we already have ways to filter data, so I'm unconvinced this functionality belongs in this component. The user can always filter a log stream before sending it through this component.

Reply:
Ahhh good catch, I had forgotten to change the description when I edited the readme.

I just updated it to be accurate with what the condition does.

The condition is actually used to determine what logs are deduped. Any logs that don't match the condition are just exported without processing.